### PR TITLE
11th Bug Crusade

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:

- Fixed issue where Mogor had two-headed trait, but looked like normal ogre.
- Fixed issue where political factions could form much later If you played with disabled defensive pacts.
- Fixed issue where corruptors of the Sunwell could change their race traits to blood elves if they were high elves.
- Fixed issue where child could be born undead if father became undead after conception.
- Fixed issue where "Shath'mag" artifact AKA Necronomicon didn't disappear after owner's death.
- Fixed issue where newly born ancient had wrong race trait.
- Fixed issue where members of factions like the Horde or Alliance continued to be allies after faction defeat.
- Fixed issue where demons could be sacrificed to Hakkar via "Hold Blood Ritual".
- Reworked "Blood and Silver" event conditions. Now it can happen if "Lordaeron" and "Stratholme" provinces aren't ruled by Light human characters. Plus, now this event gives Whispering Gardens province and Hearthglen duchy to the Scarlet Crusade.
- You no longer see who cursed you via "Bad Voodoo" decision.
- AI no longer use harmful spells against you if it has more then 50 opinion about you.
- Fixed issue where the Hand of Argus existed from the first bookmark.
- Fixed issue where "Corrupt Character" and "Attract Old Gods' Attention" could be used against characters that are immune to Fel or Void magics like demons or void beings.
- "Mind Control" and "Drive Mad" abilities no longer can be used on Lich King.
- Class level up and class change no longer changes your immortal age. So immortal females now don't get negative opinion about marriage because "they're past childbearing age".

Secret Changelog (not for patch notes):

- demesne_size replaced by global_levy_size.
- Commented out events, decisions that were connected to disabled game rules.
- Evil holy orders, adventures, mercenaries return false in can_rule_peacefully_trigger.
- Reverted accidentally deleted dragonflight icon.
- Fixed issue where patrician houses had the sames names as some human baronies.
- Fixed issue where Arthas had no education.
- Main characters in "Exodus of the Horde" or "Founding of Theramore" event chains no longer invite rulers who declined these events before them.
- Fixed issue where future Theramore leader could invite Legion worshiper.
- Fixed issue where the Scourge couldn't declare war against the united Alliance after the Plague in Lordaeron.
- Fixed issue where Horde faction could be united twice.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2138,7 +2138,7 @@ LOADING_TIP_17;Choose your mentor wisely. Your skill will depend on what the men
 LOADING_TIP_20;Lead the war very carefully, otherwise the whole world could take up arms against you.;;;;;;;;;;;;;x
 LOADING_TIP_28;Don't forget to periodically check your possible decisions. There you can find many unique features that will not always be available to you.;;;;;;;;;;;;;x
 LOADING_TIP_35;Joining a political faction is a very good way to find new allies for a very long time.;;;;;;;;;;;;;x
-LOADING_TIP_45;Watch out for the activity of dark cults in your lands, very powerful forces are behind them. If they spread their heresy, you might no longer have the chance to stop them.;;;;;;;;;;;;;x
+LOADING_TIP_45;Watch out for the activity of dark cults in your lands. If they spread their heresy, you might no longer have the chance to stop them.;;;;;;;;;;;;;x
```
Comment (DukeJeans) [inline localisation/00_zuwc_united_localisation.csv]:
```suggestion
LOADING_TIP_45;Watch out for dark cult activity in your realm. If they get the chance to spread, you may lose your only chance to stop them.;;;;;;;;;;;;;x
```

**Comment by geffe1** [comment]
>Fixed issue where Mogor had two-headed trait, but looked like normal ogre.

Fixed.

>Reworked "Blood and Silver" event conditions. Now it can happen if "Lordaeron" and "Stratholme" provinces aren't ruled by Light human characters. Plus, now this event gives Whispering Gardens province and Hearthglen duchy to the Scarlet Crusade.

Fixed, but.. yeah, no not really. I have had these scarlet events and Argent Dawn events fire multiple of times over the past 3 years in my game.

ck2_262
ck2_261

These two were the last two to fire, but the year is clearly past the point where they are meant to fire.
And here's a second one for you as an example.
ck2_263


>Fixed issue where the Hand of Argus existed from the first bookmark.

Yepp, can't find them.

**Comment by zumbak04** [comment]
> I have had these scarlet events and Argent Dawn events fire multiple of times over the past 3 years in my game.

Fixed!

**Comment by geffe1** [comment]
>Fixed issue where corruptors of the Sunwell could change their race traits to blood elves if they were high elves.

Seems to work like it should then, as I only got the event when I lost the Sunwell to become Sin'dorei.

**Comment by geffe1** [comment]
>Fixed issue where members of factions like the Horde or Alliance continued to be allies after faction defeat.

Fixed.

>Fixed issue where "Shath'mag" artifact AKA Necronomicon didn't disappear after owner's death.

Fixed.

>Fixed issue where newly born ancient had wrong race trait.

Fixed. 
ck2_270
Kawaii-desu-san.

**Pull request merged**